tests/core/test_validation_reveal_type.txt

```diff
@@ -0,0 +1,77 @@
+from typing import TYPE_CHECKING
```

Comment:
this file is valid python code, but is a .txt file instead of .py file because the linters mess up the formatting. Not sure how to make the linters ignore an entire file, but it's an option if we want to keep it as a .py file.

Reply:
updated as .py file in https://github.com/pyvista/pyvista/pull/5571/commits/e81d3774ddba8c85f77b5d8170a7b45694ea2a7e